packages/resources-manager/src/browser/resources/utils.ts

```diff
@@ -78,14 +78,19 @@ export const fetchDoor43ResourceDisplayData = async (
     [x: string]: string;
     subject: string;
     metadataType: string;
-  }
+  },
+  query?: string
```

Comment:
can you merge this `query` into the `searchParams` object instead of creating a new param